iphone/cli/commands/_build.js

```diff
@@ -3490,8 +3491,16 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 						});
 					}
 
-					if (hasSwiftFiles && !extBuildSettings.SWIFT_VERSION) {
-						extBuildSettings.SWIFT_VERSION = '2.2';
+					if (hasSwiftFiles) {
+						if (!extBuildSettings.SWIFT_VERSION) {
+							extBuildSettings.SWIFT_VERSION = '2.2';
```

Comment:
I'm not opposed to changing it, but that's outside the scope of TIMOB-23313. I'm also all for dropping Swift 1 and 2.